src/modules/resource/resource.c

```diff
@@ -183,6 +182,8 @@ static void disconnect_cb (flux_t *h,
 
     if (ctx->acquire)
         acquire_disconnect (ctx->acquire, msg);
+    if (ctx->status)
+        status_disconnect (ctx->status, msg);
```

Comment:
Uh oh just realized that `acquire_disconnect()` might segfault if called on a rank other than zero.  Will fix.

Reply:
Also realized `flux resource list` no longer works on rank > 0 and there was no test coverage! Fixups coming.